CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 3.1.1 (2019-06-10)
+
+- Fixed the flyout component to handle click events that originate within its contents. [#44](https://github.com/blackbaud/skyux-flyout/pull/44)
+- Fixed the flyout component to style the resize handle and iterator buttons correctly in Internet Explorer. [#45](https://github.com/blackbaud/skyux-flyout/pull/45)
+- Fixed the flyout component to only resize to twenty pixels less than the window width. [#46](https://github.com/blackbaud/skyux-flyout/pull/46)
+- Fixed the flyout component to use a default width of half the window size when no default width is specified by consumers. [#47](https://github.com/blackbaud/skyux-flyout/pull/47)
```

Comment:
It's just minor tweaks, but I'd change this to "Fixed the flyout component to set the default width to half of the window's width unless consumers specify a different default."

Reply:
That's fine